combat-trainer.lic

```diff
@@ -3889,7 +3891,7 @@ class GameState
 end
 
 before_dying do
-  DRCA.release_cyclics
+  DRCA.release_cyclics unless @release_cyclic
```

Comment:
This is an instance var and used outside of a class - I wouldn't have expected this to work. Did you test to make sure this change works as expected?

Reply:
I think maybe the best course is to move this to the cleanup process, as a whole. This entire thing. Know what I mean? As it stands now, it's a bit wonky to do things like this without doing things like get_settings for a single setting, etc. I could see an accessor being made for it, but I am not sure how the before_dying is handled within the scripts before the instance is deleted, etc. Does that make sense?

Reply:
I was pointed out that this is or kill-all, crazy stuff. Dunno. Maybe one of the foundational classes needs to store cleanup variables, not really sure.